docs/getting-started/logcli.md

```diff
@@ -125,6 +205,5 @@ Flags:
       --labels-length=0  Set a fixed padding to labels
 
 Args:
-  <query>    eg '{foo="bar",baz="blip"}'
-  [<regex>]
+  <query>  eg '{foo="bar",baz=~".*blip"} |~ ".*error.*"'
```

Comment:
Is this a metrics query?

Reply:
Nope, but it should be for the `instant-query` command. I'll change it, thanks.